src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -651,6 +651,14 @@ public Builder inheritProcessOutput(boolean inheritProcessOutput) {
     }
 
     /**
+     * Sets the desired Java SDK path, used in devappserver runs and App Engine standard staging.
+     */
+    public Builder javaSdkPath(Path javaSdkPath) {
```

Comment:
On the flip side, JAVA_HOME is exactly what's being done by devappserver2.  So we'll have this issue anyways.